docs/openapi-docs/src/main/scala/sttp/tapir/docs/openapi/EndpointToOpenAPIPaths.scala

```diff
@@ -44,6 +45,23 @@ private[openapi] class EndpointToOpenAPIPaths(
     (e.showPathTemplate(showQueryParam = None, includeAuth = false, showNoPathAs = "/", showPathsAs = None), pathItem)
   }
 
+  private def filterOutHiddenInputs(inputs: Vector[EndpointInput.Basic[_]]) = inputs
+    .filter {
```

Comment:
1. maybe we can combine it in a .collect?
2. are we doing filtering on hidden someplace else in this module as well? I think it used to work for e.g. queries, so maybe it's not needed now?